onnxruntime/contrib_ops/cuda/fused_conv.cc

```diff
@@ -100,6 +101,10 @@ class FusedConv : public onnxruntime::cuda::Conv<T, false> {
   Status MapMode(const std::string& activaton_mode) {
     if (activaton_mode == "Relu") {
       activation_mode_ = cudnnActivationMode_t::CUDNN_ACTIVATION_RELU;
+    } else if (activaton_mode == "Tanh") {
+      activation_mode_ = cudnnActivationMode_t::CUDNN_ACTIVATION_TANH;
```

Comment:
Can we add tests for the 2 supported activations ?

Reply:
You mean the added tests don't pass ? Unfortunately, I don't have the time to investigate this. Shall we just support these activations in another PR ?

Reply:
Yes, surely I can move them out of this PR, but the point is more that the accuracy does not pass. I would like to understand how you usually arrive at the expected values and what a typical eps would be. If you can help me out with that I'll either remove or adjust the tests.

Reply:
Based on offline sync - we will introduce the enhancements to the FusedConv in another PR